metadeploy/api/models.py

```diff
@@ -478,6 +478,14 @@ class Plan(HashIdMixin, SlugMixin, AllowedListAccessMixin, TranslatableModel):
         default=SUPPORTED_ORG_TYPES.Persistent,
     )
     org_config_name = models.CharField(max_length=64, default="release", blank=True)
+    scratch_org_duration_override = models.IntegerField(
+        "Scratch Org duration (days)",
+        null=True,
+        blank=True,
+        validators=[MinValueValidator(1)],
```

Comment:
This should have a max of 30; the platform does not support scratch orgs lasting longer than that.